javalib/src/main/scala/java/io/OutputStreamWriter.scala

```diff
@@ -45,7 +45,10 @@ class OutputStreamWriter(private[this] var out: OutputStream,
     this(out, Charset.defaultCharset)
 
   def this(out: OutputStream, charsetName: String) =
-    this(out, Charset.forName(charsetName))
+    this(out, try Charset.forName(charsetName)
+    catch {
```

Comment:
The formatting here is really weird. I suggest formatting that method as:
```scala
  def this(out: OutputStream, charsetName: String) = {
    this(out, try {
      Charset.forName(charsetName)
    } catch {
      case _: UnsupportedCharsetException =>
        throw new UnsupportedEncodingException(charsetName)
    })
  }
```

Reply:
Fixed in https://github.com/scala-js/scala-js/commit/0e5d092bae45debcfaf5e0229e7d3510281cab39